nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-client-service/pom.xml

```diff
@@ -196,7 +196,7 @@
                     <plugin>
                         <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-failsafe-plugin</artifactId>
-                        <version>3.0.0-M5</version>
+                        <version>3.0.0-M3</version>
```

Comment:
Why the downgrade here?

Reply:
Turns out there's a bug in M4/5 that prevents the classpath being populated correctly for the integration tests - i.e. they don't run with -M5, but this works with M3

Apparently it's been fixed in the latest snapshot versions of the plugin, but not yet available